pages/docs/tutorials/validate-documents.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Validate AsyncAPI document with Studio"
+description: This tutorial will teach you how to validate AsyncAPI documents using the AsyncAPI Studio tool.
+weight: 120
+---
+
+## Introduction
+This tutorial will teach you how to validate AsyncAPI documents using the [AsyncAPI Studio tool](https://studio.asyncapi.com/).
+
+You will start with a broken AsyncAPI document and troubleshoot via console errors step-by-step until we end up with a valid AsyncAPI document. This process will illustrate how to identify [`REQUIRED` properties in AsyncAPI documents](https://www.asyncapi.com/docs/reference/specification/v2.5.0#A2SObject).
+
+## Background context
+An AsyncAPI document is a file that defines and annotates the different components of a specific Event-Driven API. The format of the file must be JSON or YAML. You can use this document to generate both documentation and code.
+
+The AsyncAPI Studio tool allows you to develop an AsyncAPI document, validate it, preview it, convert it to the latest version, and visualize event flows.
+
+<Remember>
+
+If you did not follow the previous tutorial and do not have an `asyncapi.yaml` file ready, then generate one using `asyncapi new --example=tutorial.yml --no-tty` command.
+
+</Remember>
```

Comment:
btw, I saw this in at least 2 tutorials already. Might make sense to turn it into fragment too

Reply:
let's create new fragments for items longer than just 1 sentence tho